docs/app/partials/layout-children.tmpl.html

```diff
@@ -142,50 +180,81 @@
-  </p>
+
   <table class="md-api-table">
+      <thead>
+        <tr>
+          <th>API</th>
+          <th>Device <b>width</b> when breakpoint overrides default</th>
+        </tr>
+      </thead>
+       <tr>
+         <td>flex-order</td>
+         <td>Sets default layout order unless overridden by another breakpoint.</td>
+       </tr>
     <tr>
-      <td>flex-order</td>
-      <td>Sets element order.</td>
-    </tr>
-    <tr>
-      <td>flex-order-sm</td>
-      <td>Sets element order on devices less than 600px wide.</td>
-    </tr>
-    <tr>
-      <td>flex-order-gt-sm</td>
-      <td>Sets element order on devices greater than 600px wide.</td>
-    </tr>
-    <tr>
-      <td>flex-order-md</td>
-      <td>Sets element order on devices between 600px and 960px wide.</td>
-    </tr>
-    <tr>
-      <td>flex-order-gt-md</td>
-      <td>Sets element order on devices greater than 960px wide.
-    </tr>
-    <tr>
-      <td>flex-order-lg</td>
-      <td>Sets element order on devices between 960px and 1200px wide.</td>
-    </tr>
-    <tr>
-      <td>flex-order-gt-lg</td>
-      <td>Sets element order on devices greater than 1200px wide.</td>
-    </tr>
-  </table>
+        <td>flex-order-xs</td>
+           <td>width &lt; <b>600</b>px</td>
+         </tr>
+         <tr>
+           <td>flex-order-gt-xs</td>
+           <td>width &gt;= <b>600</b>px</td>
+         </tr>
+         <tr>
+           <td>flex-order-sm</td>
+           <td><b>600</b>px &lt;= width &lt; <b>960</b>px</td>
+         </tr>
+         <tr>
+           <td>flex-order-gt-sm</td>
+           <td>width &gt;= <b>960</b>px</td>
+         </tr>
+         <tr>
+           <td>flex-order-md</td>
+           <td><b>960</b>px &lt;= width &lt; <b>1280</b>px</td>
+         </tr>
+         <tr>
+           <td>flex-order-gt-md</td>
+           <td>width &gt;= <b>1280</b>px</td>
+         </tr>
+         <tr>
+           <td>flex-order-lg</td>
+           <td><b>1280</b>px &lt;= width &lt; <b>1920</b>px</td>
+         </tr>
+         <tr>
+           <td>flex-order-gt-lg</td>
+           <td>width &gt;= <b>1920/b>px</td>
+         </tr>
+         <tr>
+           <td>flex-order-xl</td>
+           <td>width &gt;= <b>1920</b>px</td>
+         </tr>
+     </table>
+  
+
   <p>
     See the <a href="layout/options">layout options page</a> for more information on attributes like
     <code>hide</code>, <code>hide-gt-md</code>, and <code>show-gt-md</code> used in the above examples.
   </p>
 
+  <br/>
+  <hr>
+  <br/>
+
+  <h3>Add Offsets to the Preceding HTML Elements</h3>
+
+  <p>
+    Add the <code>flex-offset</code> attribute to a layout child to set its
+    offset percentage within the layout container. Values must be multiples
+    of <code>5</code> or <code>33</code> / <code>66</code>. These offsets establish a <code>margin-left</code>
+    with respect to the preceding element or the containers left boundary.
+  </p>
+
```

Comment:
Might want to add a note here that this doesn't automatically switch to the right side if you use `flex-order` or manually set the direction to `flex-reverse`.